taichi/codegen/llvm/codegen_llvm.h

```diff
@@ -138,6 +138,15 @@ class TaskCodeGenLLVM : public IRVisitor, public LLVMModuleBuilder {
 
   llvm::Value *create_print(std::string tag, llvm::Value *value);
 
+  void create_return(const std::vector<Stmt *> &elements);
+
+  void create_return(llvm::Value *buffer,
```

Comment:
Looks like this overloaded `create_return()` is an implementation of `void create_return(const std::vector<Stmt *> &elements);`, which won't get called by other compile unit? If that is the case, let's make it a static function in `codegen_llvm.cpp`?

Reply:
Do you mean private function? `create_return` uses a lot of non-static members of class `CodeGenLLVM` so it's not possible to be a static function.

Reply:
yep I meant `private funtion`